unit_tests/test_scylla_yaml_builders.py

```diff
@@ -552,6 +555,7 @@ def _run_test(self, config_path: str, expected_node_config: str):
                 os.environ['SCT_CLUSTER_BACKEND'] = 'aws'
                 os.environ['SCT_REGION_NAME'] = '["eu-west-1", "us-east-1"]'
                 os.environ['SCT_CONFIG_FILES'] = config_path
+                os.environ['SCT_PREPARE_SASLAUTHD'] = "true"
```

Comment:
this is a bit lazy, you should have changed the json expected files...

but never mind...